src/lib/select/select.spec.ts

```diff
@@ -396,6 +395,7 @@ describe('MatSelect', () => {
         it('should open the panel when pressing a vertical arrow key on a closed multiple select',
           fakeAsync(() => {
             fixture.destroy();
```

Comment:
We should have a test that verifies that the select panel is in the DOM and with the correct amount of options on init.